misk-aws/src/main/kotlin/misk/dynamodb/DynamoDbHelpers.kt

```diff
@@ -0,0 +1,42 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import java.time.Duration
+import misk.backoff.Backoff
+import misk.backoff.DontRetryException
+import misk.backoff.ExponentialBackoff
+import misk.backoff.retry
+
+object DynamoDbHelpers {
+  fun getDefaultBackoff(): ExponentialBackoff {
+    return ExponentialBackoff(
+        baseDelay = Duration.ofMillis(10),
```

Comment:
Do these seem reasonable? They're not based on any hard numbers

Reply:
I would put `100ms`

Reply:
Not sure how trivial is coming up with a good number.